app/clients/chrome_client.rb

```diff
@@ -0,0 +1,26 @@
+require 'selenium-webdriver'
+
+class ChromeClient
+  CHROME_CLIENT_TIMEOUT = 35
+  ENABLE_HEADLESS_CHROME = true
+
+  @@options = Selenium::WebDriver::Chrome::Options.new
+
+  def browser
+    if ENABLE_HEADLESS_CHROME
+      set_headless_chrome
+    end
+    @browser ||= Pincers.for_webdriver(:chrome, wait_timeout: CHROME_CLIENT_TIMEOUT, options: @@options)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/LineLength: Line is too long. [104/100]